config/environments/test.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Aj tu je to divne, asi do application.rb?

Reply:
Railsy to pridavaju
https://stackoverflow.com/questions/67649077/why-rails6-started-adding-activesupport-requires-in-config-environments-by-de